code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -838,3 +838,13 @@
 	name = "AB- blood patch"
 	desc = "An embroidered patch indicating the wearer's blood type as AB NEGATIVE."
 	icon_state = "abnegtag"
+
+/obj/item/clothing/accessory/IPC_Mantle
+	name = "\improper Burzsiean shoulder mantle"
+	desc = "An uniform mantle pulled over the right shoulder. The sigil of Burzsia is pressed into the front of it."
+	desc_fluff = "An identifying tarp issued to industrial IPC's that work without protective equipment on Burzsia. Tear resistant and chemically treated to withstand the planet of Burzsia I. More commonly seen on industrials or units that otherwise work inside of installations. Operation history and specifications are printed underneath the tarp, for field operators to quickly identify the unit in the event it is damaged to the point where it cannot give diagnostics."
```

Comment:
```suggestion
	desc_fluff = "An identifying tarp issued to industrial IPCs that work without protective equipment on Burzsia. Tear resistant and chemically treated to withstand the planet of Burzsia I. More commonly seen on industrials or units that otherwise work inside of installations. Operation history and specifications are printed underneath the tarp, for field operators to quickly identify the unit in the event it is damaged to the point where it cannot give diagnostics."
```